LICENSE-binary

```diff
@@ -516,6 +522,7 @@ Common Development and Distribution License (CDDL) 1.1
 
 javax.el:javax.el-api	https://javaee.github.io/uel-ri/
 javax.servlet:javax.servlet-api   https://javaee.github.io/servlet-spec/
+javax.servlet.jsp:jsp-api
```

Comment:
For super completeness, let's put a copy of its license in `license-binary/`

Reply:
> For super completeness

Add this 
https://github.com/javaee/javaee-jsp-api/blob/master/LICENSE